che-theia-java-extension/src/browser/navigation/find-implementers.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which is available at http://www.eclipse.org/legal/epl-2.0.html
```

Comment:
FYI seems the link is failing for EPL 2 but maybe it's because you used a copy/paste from Eclipse Che which had the same issue ?